modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/atomic/GridNearAtomicSingleUpdateFuture.java

```diff
@@ -448,6 +456,8 @@ private void updateNear(GridNearAtomicAbstractUpdateRequest req, GridNearAtomicU
 
     /** {@inheritDoc} */
     @Override protected void map(AffinityTopologyVersion topVer) {
+        MTC.span().addTag("topology.version", () -> Objects.toString(topVer));
```

Comment:
Why add to thread-local span instead of this.span?

Reply:
Also actual for GridNearAtomicUpdateFuture

Reply:
Fixed.